### ISSUE TITLE
Deduplicate - Find missing element

### ISSUE THREAD

**Issue by hobovsky** [codewars/content-issues#184]
From #39

1. [Return the Missing Element](https://www.codewars.com/kata/5299413901337c637e000004)
* 7kyu
* input: array of numbers from 0-9 with one missing
* Satisfaction: 94% with 9k completions
* No pending issues
* 10 languages + 2 pending translations, 4 unique
* Published Nov 2013, author inactive

2. [Find the Missing Number](https://www.codewars.com/kata/57f5e7bd60d0a0cfd900032d)
* 7kyu
* input: array of numbers from 0-100(inclusive) with one missing)
* Satisfaction: 93% with 5k solutions
* 1 pending issue
* 10 languages + 3 pending translations, 5 unique
* Published Oct 2016, author inactive

### Conclusion

- Keep 1. [Return the Missing Element](https://www.codewars.com/kata/5299413901337c637e000004)
- [ ] Retire 2. [Find the Missing Number](https://www.codewars.com/kata/57f5e7bd60d0a0cfd900032d)
  - [ ] Move Clojure translation
  - [ ] Move COBOL translation
  - [ ] Move Elixir translation
  - [ ] Move Java translation
  - [ ] Move Kotlin translation

**Comment by hobovsky** [comment]
I have no good candidate here. Kata 1. is older and has more completions, but looks close to trivial. But still, range of 2. does not really make any difference. Maintenance-wise they seem to be similar.

Any ideas?

**Comment by Blind4Basics** [comment]
keep 2, retire 1. The amount of work seems slightly lower with 2 (one less translation/one less unique)

**Comment by JohanWiltink** [comment]
They're functionally identical. (2) is as trivial as (1).

Keep 1, retire 2. The newer one should not have been approved, as a duplicate, and no need to retire _more_ solutions.

**Comment by Blind4Basics** [comment]
solutions are not retired in any way. That's a non argument ;)

**Comment by 4500zenja1** [comment]
Since these katas are almost identical to each other with the same approach, I suggest to **keep the oldest one** (namely, 1) and **retire 2** with transferring all existing unique languages prior that (these are, Clojure, COBOL, C++, Elixir).

**Comment by ejini6969** [comment]
This one can be concluded too

**Comment by hobovsky** [comment]
Yes, I think that with just one outlier vote we can conclude this one as "keep 1., retire 2."
